surface/include/pcl/surface/3rdparty/poisson4/bspline_data.hpp

```diff
@@ -443,8 +444,8 @@ namespace pcl
     template< int Degree >
     void BSplineElements< Degree >::upSample( BSplineElements< Degree >& high ) const
     {
-      fprintf( stderr , "[ERROR] B-spline up-sampling not supported for degree %d\n" , Degree );
-      exit( 0 );
+      fprintf (stderr, "[ERROR] B-spline up-sampling not supported for degree %d\n", Degree);
```

Comment:
Let me know if you want me to remove these fprintf stderr log messages

Reply:
I'd :+1: removal of `stderr` messages. Now that there are exceptions, the user can decide what to do. We will miss a bit of debug information around line 230, but I don't think it's important for a user anyway.

Reply:
removed in abbe9538476fd6c44d3563551439a7656b41239f

Reply:
> We will miss a bit of debug information around line 230, but I don't think it's important for a user anyway.

Well, not necessarily. Exceptions do support messages.

Reply:
Before, lines ~230 were logging detailed info about each "unindexed corner" (actually only the first.. because of the `exit( 0 );` )

Now it's replaced with a single exception message that only reports a count e.g. `Found 37 unindexed corner nodes during openMP loop execution.` without any info about the specific corners involved.

Are you proposing we build up a string that includes details about every unindexed corner?

Reply:
I spoke too soon. I agree with your call. It's too much info to put on an exception.